testing/python/fixtures.py

```diff
@@ -622,16 +640,16 @@ def test_func(something): pass
         """
         )
         req = fixtures.FixtureRequest(item, _ispytest=True)
-        assert req.function == item.obj
+        assert req.function == item.obj  # type: ignore[attr-defined]
```

Comment:
At this point you probably already know what I'm going to say :) Basically for all of the type-ignores, it would be better to use an assert instead, unless it's not possible.

Reply:
added assertions